controllers/base.go

```diff
@@ -0,0 +1,74 @@
+package controllers
+
+import (
+    "net/http"
+    "gopkg.in/macaron.v1"
+)
+
+var (
+    SPONSORS []Sponsor = []Sponsor{
```

Comment:
Correct me if I'm wrong but I think Go's convention is `Sponsors`, not full caps